client/network/src/protocol/sync.rs

```diff
@@ -620,6 +687,17 @@ impl<B: BlockT> ChainSync<B> {
 					return Ok(None)
 				}
 
+				if let SyncMode::Warp = &self.mode {
+					// TODO: wait for 3 peers
```

Comment:
?

Reply:
I've removed the check for local testing. Will put it back.